09Ianuarie/src/ro/sci/ale/ian09/max/MaxNumber.java

```diff
@@ -0,0 +1,49 @@
+package ro.sci.ale.ian09.max;
+
+import java.util.Scanner;
+
+/**
+ * implements an application that reads three numbers from the console
+ * and prints the biggest of them
+ *
+ * @author Buda Alexandra
+ */
+public class MaxNumber {
+    /**
+     * Use the getMax() method and prints
+     * the biggest number
+     *
+     * @param args unused.
+     */
+    public static void main(String[] args) {
+        Scanner read = new Scanner(System.in);
+
+        System.out.println("Enter the first number: ");
+        int number1 = read.nextInt();
+
+        System.out.println("Enter the second number: ");
+        int number2 = read.nextInt();
+
+        System.out.println("Enter the third number: ");
+        int number3 = read.nextInt();
+
+        int biggest = Math.max(getMax(number1, number2), number3);
```

Comment:
why use Math.max when you can use getMax twice?
getMax(getMax(number1, number2), number3);